src/adhocracy_frontend/adhocracy_frontend/static/js/mkResources.ts

```diff
@@ -258,177 +257,19 @@ compileAll = (metaApi : MetaApi.IMetaApi, outPath : string) : void => {
-            s += "        // FIXME: workaround for #261.  Remove if ticket is closed.\n";
-            s += "        _.forOwn(args, (value, key) => {\n";
-            s += "            if (!_.includes(" + JSON.stringify(sheet.fields.map((fieldName) => fieldName.name)) + ", key)) {\n";
-            s += "                this[key] = value;\n";
-            s += "            }\n";
-            s += "        });\n";
-        }
-
-        s += "    }\n";
-        return s;
-    };
-
-    var mkParse = () => {
-        var args : string[] = [];
-        var lines : string[] = [];
-
-        if (sheet.fields.length > 0) {
-            args.push("args : {");
-            args.push(mkFieldSignaturesSheetParse(sheet.fields, "        ", ";\n") + ";");
-            args.push("    }");
-
-            lines.push("        var parsedArgs = {");
-            for (var x in sheet.fields) {
-                if (sheet.fields.hasOwnProperty(x)) {
-                    var codeLine : string;
-                    var fieldName : string = sheet.fields[x].name;
-                    var fieldParser : string = mkFieldType(sheet.fields[x]).parser;
-
-                    if (fieldParser) {
-                        codeLine = "(" + fieldParser + ")(args." + fieldName + ")";
-                    } else {
-                        codeLine = "args." + fieldName;
-                    }
-
-                    if (!isWriteableField(sheet.fields[x])) {
-                        codeLine = "args.hasOwnProperty(\"" + fieldName + "\") ? " + codeLine + " : undefined";
-                    }
-
-                    lines.push("            " + fieldName + ": " + codeLine + ",");
-                }
-            }
-            lines.push("        };");
-        }
-
-        var s = "";
-        s += "    static parse(" + args.join("\n") + ") {\n";
-        if (lines.length > 0) {
-            s += lines.join("\n") + "\n";
-        }
-
-        // FIXME: workaround for #261.  Remove if ticket is closed.
-        if (sheet.fields.length > 0) {
-            s += "\n";
-            s += "        // FIXME: workaround for #261.  Remove if ticket is closed.\n";
-            s += "        _.forOwn(args, (value, key) => {\n";
-            s += "            if (!_.includes(" + JSON.stringify(sheet.fields.map((fieldName) => fieldName.name)) + ", key)) {\n";
-            s += "                parsedArgs[key] = value;\n";
-            s += "            }\n";
-            s += "        });\n";
-        }
-
-        if (sheet.fields.length > 0) {
-            s += "        return new Sheet(parsedArgs);\n";
-        } else {
-            s += "        return new Sheet();\n";
-        }
-
-        s += "    }\n";
-        return s;
-    };
-
-    var showList = (elems : string[]) : string => {
-        if (elems.length === 0) {
-            return "[]";
-        } else {
-            return "[\"" + elems.join("\", \"") + "\"]";
-        }
-    };
-
-    sheetI += "import * as _ from \"lodash\";\n\n";
```

Comment:
Ok I have a good diea of what this does. I've also found a redundant arg; I'll push a fix soon.